### ISSUE TITLE
WCAG (2.4.2): Add proper page titles

### ISSUE THREAD

**Issue by melaniekung** [artefactual/atom#1673]
[WCAG 2.4.2 Page Titled](https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html) states that web pages should have titles that describe the topic or purpose.

**Comment by melaniekung** [comment]
PR: https://github.com/artefactual/atom/commit/60c961c37b9d24ffabe9dbb5aca38a7e886b1d61
